site/content/docs/background/architecture.md

```diff
@@ -90,14 +92,13 @@ cleanly enable this integration.
 
 Pinniped supports the following cluster integration strategies.
```

Comment:
Should this "Cluster IntegrationStrategies" section mirror what is served by the `CredentialIssuer.Status.Strategies` API: https://github.com/vmware-tanzu/pinniped/tree/main/generated/1.20#k8s-api-go-pinniped-dev-generated-1-20-apis-concierge-config-v1alpha1-credentialissuerstrategy? I'm wondering if the "Cluster IntegrationStrategies" are more like `KubeClusterSigningCertificate`/`ImpersonationProxy` instead of `TokenCredentialRequestAPI`/`ImpersonationProxy` (which are "frontends"). I came upon this thought when I was thinking about how the `TokenCredentialRequestAPI` is actually used in both integration strategies, just in different ways.

WDYT?

Reply:
Yeah, I was asking myself the same thing... `KubeClusterSigningCertificate` might be more accurate, but it's also kind of a weird hack that we don't want to put front and center... But then again, the impersonation proxy is kind of a weird hack as well, because we can't get the CSR API to do exactly what we want. 
I kind of like `TokenCredentialRequestAPI` because the token credential request is the thing doing the heavy lifting. Even if it's also used elsewhere for convenience.